### PR TITLE
CP-9266: use forked react-native-webview package

### PR DESCRIPTION
**Pull request by ruijialin-avalabs**
## Description

**Ticket: [CP-9266]** 

- always prompt permission dialog when a website is requesting camera and file download permission
- custom permission prompt is displayed before the system permission prompt

## ava-labs/react-native-webview branch
https://github.com/ava-labs/react-native-webview/tree/cp-9266

## Screenshots/Videos

camera permission request

https://github.com/user-attachments/assets/87c0635e-cb36-4aec-9521-9af0b64b0fab

file download permission request

https://github.com/user-attachments/assets/066a0b50-e01c-48a4-90d2-f5d36fa946e4

## Checklist

Please check all that apply (if applicable)
- [X] I have performed a self-review of my code
- [X] I have verified the code works
- [ ] I have added/updated necessary unit tests 
- [ ] I have updated the documentation

[CP-9266]: https://ava-labs.atlassian.net/browse/CP-9266?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .yarnrc.yml
Comment (ruijialin-avalabs) [inline .yarnrc.yml:9]: Had to add this to fix the cached package issue in CI.

`The remote archive doesn't match the expected checksum`

Comment (ruijialin-avalabs) [reply .yarnrc.yml:9]: I guess we can remove it in another pr once the issue above is fixed in ci.

**Review by atn4z7 — APPROVED** [review]
let's wait on merging this until we get an approval from security team

**Review by chriselbring-avalabs — APPROVED** [review]
LGTM! 

Very nice, thank you for the quick turnaround on this!

**Pull request merged**
